web/src/contexts/amplitude-context.jsx

```diff
@@ -13,13 +15,15 @@ export const AmplitudeContextProvider = ({ children }) => {
   const ref = useRef(false);
   const [amplitude, setAmplitude] = useState(null);
 
+  const { data: currentAdmin } = adminApi.useGetCurrent();
+
   useEffect(() => {
     if (router.isReady && !ref.current) {
       ref.current = true;
-      Amplitude.init(publicRuntimeConfig.NEXT_PUBLIC_AMPLITUDE_API_KEY);
+      Amplitude.init(publicRuntimeConfig.NEXT_PUBLIC_AMPLITUDE_API_KEY, currentAdmin?._id, { plan: { branch: 'development' } });
```

Comment:
Is it always `development` for a `branch` field?

Reply:
no, i need to fix it